ql/src/test/results/clientpositive/llap/stats_part.q.out

```diff
@@ -161,7 +161,7 @@ Table Parameters:
 	numPartitions       	0                   
 	numRows             	0                   
 	rawDataSize         	0                   
-	totalSize           	#Masked#
+	totalSize           	#Masked#                   
```

Comment:
Need to regenerate this out once remove the `totalSize`.

Reply:
Done, thanks